bundle/manifests/nic-configuration-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,373 @@
+                "maxAccOutRead": 44,
+                "maxReadRequest": 4096
+              },
+              "roceOptimized": {
+                "enabled": true,
+                "qos": {
+                  "pfc": "0,0,0,1,0,0,0,0",
+                  "trust": "dscp"
+                }
+              }
+            }
+          }
+        },
+        {
+          "apiVersion": "configuration.net.nvidia.com/v1alpha1",
+          "kind": "NicDevice",
+          "metadata": {
+            "name": "co-node-25-101b-mt2232t13210",
+            "namespace": "nic-configuration-operator"
+          },
+          "spec": {
+            "configuration": {
+              "template": {
+                "linkType": "Ethernet",
+                "numVfs": 8,
+                "pciPerformanceOptimized": {
+                  "enabled": true
+                }
+              }
+            }
+          },
+          "status": {
+            "conditions": [
+              {
+                "reason": "UpdateSuccessful",
+                "status": "False",
+                "type": "ConfigUpdateInProgress"
+              }
+            ],
+            "firmwareVersion": "20.42.1000",
+            "node": "co-node-25",
+            "partNumber": "mcx632312a-hdat",
+            "ports": [
+              {
+                "networkInterface": "enp4s0f0np0",
+                "pci": "0000:04:00.0",
+                "rdmaInterface": "mlx5_0"
+              },
+              {
+                "networkInterface": "enp4s0f1np1",
+                "pci": "0000:04:00.1",
+                "rdmaInterface": "mlx5_1"
+              }
+            ],
+            "psid": "mt_0000000225",
+            "serialNumber": "mt2232t13210",
+            "type": "101b"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    createdAt: "2024-12-20T11:10:42Z"
+    operators.operatorframework.io/builder: operator-sdk-v1.37.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v4
+  name: nic-configuration-operator.v1.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: NicConfigurationTemplate is the Schema for the nicconfigurationtemplates
+        API
+      displayName: Nic Configuration Template
+      kind: NicConfigurationTemplate
+      name: nicconfigurationtemplates.configuration.net.nvidia.com
+      version: v1alpha1
+    - description: NicDevice is the Schema for the nicdevices API
+      displayName: Nic Device
+      kind: NicDevice
+      name: nicdevices.configuration.net.nvidia.com
+      version: v1alpha1
+  description: NVIDIA NIC Configuration Operator provides Kubernetes API (Custom Resource
+    Definition) to allow FW configuration on NVIDIA NICs in a coordinated manner.
+    It deploys a configuration daemon on each of the desired nodes to configure NVIDIA
+    NICs there.
+  displayName: NVIDIA NIC Configuration Operator
+  icon:
+  - base64data: ""
```

Comment:
We should have an icon.

For example:
https://github.com/redhat-openshift-ecosystem/certified-operators/blob/main/operators/nvidia-network-operator/24.1.0/manifests/nvidia-network-operator.clusterserviceversion.yaml#L132

Reply:
We should probably request one from the product designers (I guess). Let's skip it for now and add once available to us before creating a PR to certified operators

Reply:
Ack.
Maybe @e0ne remembers how we got one for Network Operator